homeassistant/components/renson_endura_delta/sensor.py

```diff
@@ -0,0 +1,269 @@
+                rensonApi,
+                False,
+            ),
+            RensonBinarySensor(
+                FROST_PROTECTION_DESC,
+                FROST_PROTECTION_FIELD,
+                rensonApi,
+            ),
+            SensorValue(
+                MANUAL_LEVEL_DESC,
+                MANUAL_LEVEL_FIELD,
+                rensonApi,
+                False,
+            ),
+            SensorValue(
+                TIME_AND_DATE_DESC,
+                TIME_AND_DATE_FIELD,
+                rensonApi,
+                False,
+            ),
+            SensorValue(
+                BREEZE_TEMPERATURE_DESC,
+                BREEZE_TEMPERATURE_FIELD,
+                rensonApi,
+                False,
+            ),
+            RensonBinarySensor(
+                BREEZE_ENABLE_DESC,
+                BREEZE_ENABLE_FIELD,
+                rensonApi,
+            ),
+            SensorValue(
+                BREEZE_LEVEL_DESC,
+                BREEZE_LEVEL_FIELD,
+                rensonApi,
+                False,
+            ),
+            RensonBinarySensor(
+                BREEZE_MET_DESC,
+                BREEZE_MET_FIELD,
+                rensonApi,
+            ),
+            SensorValue(
+                DAYTIME_DESC,
+                DAYTIME_FIELD,
+                rensonApi,
+                False,
+            ),
+            SensorValue(
+                NIGHTTIME_DESC,
+                NIGHTTIME_FIELD,
+                rensonApi,
+                False,
+            ),
+            SensorValue(
+                DAY_POLLUTION_DESC,
+                DAY_POLLUTION_FIELD,
+                rensonApi,
+                False,
+            ),
+            SensorValue(
+                NIGHT_POLLUTION_DESC,
+                NIGHT_POLLUTION_FIELD,
+                rensonApi,
+                False,
+            ),
+            RensonBinarySensor(
+                HUMIDITY_CONTROL_DESC,
+                HUMIDITY_CONTROL_FIELD,
+                rensonApi,
+            ),
+            RensonBinarySensor(
+                AIR_QUALITY_CONTROL_DESC,
+                AIR_QUALITY_CONTROL_FIELD,
+                rensonApi,
+            ),
+            RensonBinarySensor(
+                CO2_CONTROL_DESC,
+                CO2_CONTROL_FIELD,
+                rensonApi,
+            ),
+            SensorValue(
+                CO2_THRESHOLD_DESC,
+                CO2_THRESHOLD_FIELD,
+                rensonApi,
+                False,
+            ),
+            SensorValue(
+                CO2_HYSTERESIS_DESC,
+                CO2_HYSTERESIS_FIELD,
+                rensonApi,
+                False,
+            ),
+            RensonBinarySensor(
+                PREHEATER_DESC,
+                PREHEATER_FIELD,
+                rensonApi,
+            ),
+            SensorValue(
+                BYPASS_TEMPERATURE_DESC,
+                BYPASS_TEMPERATURE_FIELD,
+                rensonApi,
+                False,
+            ),
```

Comment:
You can extend the description class to add your own fields so you can merge `_DESC` and `_FIELD` constants.

Reply:
And `rawValue` can also be stored in there.